examples/java/hello_world/MainActivity.java

```diff
@@ -60,6 +60,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     engine = new AndroidEngineBuilder(getApplication())
                  .addLogLevel(LogLevel.DEBUG)
+                 .addStatsDomain("0.0.0.0")
```

Comment:
We don't need to add this. It actually slows example app startup because the DNS resolver blocks on trying to resolve "0.0.0.0" - which is neither a valid IP address, nor a valid domain name - and inevitably fails while the rest of the app waits around.

Reply:
oh right. updated